DistributedLock.Redis/RedisDistributedSynchronizationProvider.cs

```diff
@@ -26,6 +26,33 @@ public RedisDistributedSynchronizationProvider(IDatabase database, Action<RedisD
         {
         }
 
+#if NETSTANDARD2_0_OR_GREATER
+        /// <summary>
+        /// Constructs a <see cref="RedisDistributedSynchronizationProvider"/> that connects to redis according to the provided <paramref name="connectionString"/>
+        /// and the <paramref name="db"/> with the provided <paramref name="options"/>.
+        /// </summary>
+        /// <param name="connectionString">Redis connection string</param>
+        /// <param name="db">Redis Database</param>
+        /// <param name="options">Redis Distributed Synchronization Options</param>
+        public RedisDistributedSynchronizationProvider(string connectionString, int db = -1, Action<RedisDistributedSynchronizationOptionsBuilder>? options = null)
+            : this(ConfigurationOptions.Parse(connectionString), db, options)
```

Comment:
Does this add anything new in terms of capabilities or is it just convenience? Seems to me like someone who wanted this could replicate the functionality through a static factory method:

```
public static RedisDistributedSynchronizationProvider CreateProvider(string connectionString, ...) =>
    new RedisDistributedSynchronizationProvider(ConnectionMultiplexer.Connect(ConfigurationOptions.Parse(connectionString)).GetDatabase(db), ...);
```
```